docs/API.md

```diff
@@ -796,11 +797,20 @@ __Parameters__
 | ``prefix`` | _String_ | Only listen for objects with the given prefix. |
 | ``suffix`` | _String_ | Only listen for objects with the given suffix. |
 | ``events`` | _String[]_ | Only listen for the specified events, such as s3:ObjectCreated:*, s3:ObjectAccessed:*, s3:ObjectRemoved:*, ..  |
+| ``listener`` | _ BucketEventListener_ | Interface with updateEvent method |
 
 | Return Type	  | Exceptions	  |
 |:--- |:--- |
 |  None  | Listed Exceptions: |
-|        |  ``RuntimeException`` : runtime exception. |
+|        |  ``InvalidBucketNameException`` : upon invalid bucket name. |
+|        | ``NoSuchAlgorithmException`` : upon requested algorithm was not found during signature calculation.  |
+|        | ``InsufficientDataException`` : Thrown to indicate that reading given InputStream gets EOFException before reading given length. |
```

Comment:
I guess you need a space in between `EOF` and `Exception`

Reply:
modified